package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rainbird-standard-agent",
-  "version": "2.16.3",
+  "version": "2.17.1",
```

Comment:
Hi Lawrie, should this be 2.16.4?
Thanks.

Reply:
Yea, sorry I got them the wrong way round!